catalog/gke/cluster/cluster.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     gke.io/environment: dev # kpt-set: ${environment}
```

Comment:
Users could use https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/examples/set-labels-simple to set a label. We could remove this setter.

Reply:
removed the env setter as well as label since the function can insert as well